.environment/pre-commit.runner.sh

```diff
@@ -14,8 +14,8 @@ pushd "${REPO_ROOT?}" 1>/dev/null 2>&1
 CHECKS_TO_RUN=(
     #${REPO_ROOT}/.environment/gitleaks/run-gitleaks.sh
     ${REPO_ROOT}/.environment/ktlint/run-ktlintCheck.sh
-    ${REPO_ROOT}/.environment/terraform-fmt/run-terraform-fmt.sh
-    ${REPO_ROOT}/.environment/frontend/run-frontend.sh
+    #${REPO_ROOT}/.environment/terraform-fmt/run-terraform-fmt.sh
+    #${REPO_ROOT}/.environment/frontend/run-frontend.sh
```

Comment:
Please restore these test.  Both should short-circuit and exit quickly if you have no relevant changes.

Reply:
Thanks for the callout! Didnt intend to leave these committed they were temporary following a merge.